python/dgl/nn/pytorch/conv/sageconv.py

```diff
@@ -266,12 +254,12 @@ def forward(self, graph, feat, edge_weight=None):
             # GraphSAGE GCN does not require fc_self.
             if self._aggre_type == 'gcn':
                 rst = h_neigh
+                # add bias manually for GCN
+                if self.bias is not None:
+                    rst = rst + self.bias
             else:
                 rst = self.fc_self(h_self) + h_neigh
 
-            # bias term
-            if self.bias is not None:
-                rst = rst + self.bias
```

Comment:
We are only adding bias when the aggregation type is GCN?  This doesn't look right.

Reply:
No - for GCN fc_self linear is not performed, but BIAS is added anyway - for other aggregation types fc_self is performed in line 261 and BIAS is added within this Linear layer
`rst = self.fc_self(h_self) + h_neigh` ==> `(h_self * WEIGHTS + BIAS) + h_neigh`